.travis.yml

```diff
@@ -41,6 +44,7 @@ before_script:
  - php ~/travis-support/travis_setup_php54_webserver.php --if-env BEHAT_TEST
 
 script:
+ - npm run sanity
```

Comment:
Hmmm it's not really related to Behat but I see what you mean. Maybe a new variable `JAVASCRIPT_TESTS` (or something similar) which runs the sanity check, and when we have them, JS unit tests?